src/java/org/apache/cassandra/net/OutboundConnectionSettings.java

```diff
@@ -508,20 +500,4 @@ static boolean shouldCompressConnection(IEndpointSnitch snitch, InetAddressAndPo
                || ((DatabaseDescriptor.internodeCompression() == Config.InternodeCompression.dc) && !isInLocalDC(snitch, localHost, remoteHost));
     }
 
-    private static InetAddressAndPort maybeWithSecurePort(InetAddressAndPort address, int messagingVersion, boolean isEncrypted)
-    {
-        if (!isEncrypted || messagingVersion >= VERSION_40)
```

Comment:
Nit: this leaves an unused import of `MessagingService.VERSION_40` that will make the build fail.

Reply:
done